tests/ssl-opt.sh

```diff
@@ -634,6 +635,15 @@ detect_dtls() {
     fi
 }
 
+# check if the given command uses dtls and sets global variable DTLS
+detect_gnutls_serv() {
+    if echo "$1" | grep 'gnutls-serv \|gnutls-serv$' >/dev/null; then
```

Comment:
Please favor `case` and other shell built-in features over external calls such as `grep`. It's noticeably faster when running a small number of test cases.